lib/axe-injector.js

```diff
@@ -78,7 +79,7 @@ class AxeInjector {
     await this.driver.executeScript(this.script);
 
     // Get all of <iframe>s at this level
-    const frames = await this.driver.findElements({ tagName: 'iframe' });
+    const frames = await this.driver.findElements(By.tagName('iframe'));
```

Comment:
While I think this is a good improvement, since this is idiomatic use of Selenium-webdriver, it seems to me this is functionally the same (https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/by.js#L275). Can you explain how this fixes the "failed to inject" issue?